server/models.py

```diff
@@ -45,7 +45,7 @@ class Election(BaseModel):
     random_seed = Column(String(100))
 
     # an election is "online" if every ballot is entered online, vs. offline in a tally sheet.
-    online = Column(Boolean, nullable=False, default=False)
```

Comment:
shouldn't this come with an alembic migration?

Reply:
i thought so, but apparently the default value is handled by SQLAlchemy, not postgres, so there's nothing to migrate

Reply:
there's another option `server_default` that does it at the db level

Reply:
ooooh, bad SQLAlchemy, bad. Anyways, that's fine then!